torch/_inductor/compile_fx.py

```diff
@@ -1079,6 +1087,10 @@ def bw_compiler(model: torch.fx.GraphModule, example_inputs):
     tracing_context = (
         torch._guards.TracingContext.get() or torch._guards.TracingContext(fake_mode)
     )
+    if _in_aot_compilation:
+        with V.set_fake_mode(fake_mode), compiled_autograd.disable():
+            return fw_compiler(model_, example_inputs_)
```

Comment:
This is skipping freezing because we changed it to invoking `fw_compiler` instead of `inference_compiler`

Reply:
I have a followup PR to change this to `inference_compiler`: https://github.com/pytorch/pytorch/pull/106239, but right now freezing doesn't work with the c++ wrapper so I wasn't sure if I should make that change here.

Reply:
sounds good